provisioning/provisioning-service-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/service/configs/ClientCertificateIssuancePolicy.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.sdk.iot.provisioning.service.configs;
+
+import java.security.PrivateKey;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public class ClientCertificateIssuancePolicy
+{
+    // The CA name that can issue an operational certificate for a device on behalf of DPS.
+    @Getter
+    @Setter
+    public PrivateKey CertificateAuthorityName;
```

Comment:
The type and the variable name seem a bit incongruous here. A certificate authority name is conceptually just a string whereas a PrivateKey is a lot more than that. Is this based on the work we did in C#?